examples/bots/discord-bot/src/main.rs

```diff
@@ -0,0 +1,170 @@
+use clap::Parser;
+use poise::serenity_prelude::{self as serenity, Mentionable};
+use reqwest::Url;
+use some_verifier_lib::{Platform, Verification};
+
+#[derive(clap::Parser, Debug)]
+#[clap(arg_required_else_help(true))]
+#[clap(version, author)]
+struct App {
+    #[clap(
+        long = "token",
+        help = "Discord bot API token.",
+        env = "DISCORD_BOT_TOKEN"
+    )]
+    bot_token: String,
+    #[clap(
+        long = "log-level",
+        default_value = "info",
+        help = "Maximum log level.",
+        env = "DISCORD_BOT_LOG_LEVEL"
+    )]
+    log_level: tracing_subscriber::filter::LevelFilter,
+    #[clap(
+        long = "verifier-url",
+        default_value = "http://127.0.0.1:8080/",
+        help = "URL of the SoMe verifier.",
+        env = "SOME_VERIFIER_URL"
+    )]
+    verifier_url: Url,
+}
+
+struct BotConfig {
+    verifier_url: Url,
+    client: reqwest::Client,
+}
+type Context<'a> = poise::Context<'a, BotConfig, anyhow::Error>;
+
+// Note: The doc comments of the commands below define what the user sees as a help message
+/// Displays the list of commands
+#[poise::command(slash_command, prefix_command)]
+async fn help(
+    ctx: Context<'_>,
+    #[description = "Specific command to show help about"]
+    #[autocomplete = "poise::builtins::autocomplete_command"]
+    command: Option<String>,
+) -> anyhow::Result<()> {
+    poise::builtins::help(
+        ctx,
+        command.as_deref(),
+        poise::builtins::HelpConfiguration::default(),
+    )
+    .await?;
+    Ok(())
+}
+
+/// Verify with Concordium
+#[poise::command(slash_command, prefix_command)]
+async fn verify(ctx: Context<'_>) -> anyhow::Result<()> {
+    ctx.send(|reply| {
+        reply
+            .content("Please verify with your wallet.")
+            .components(link_button(&ctx.data().verifier_url, "Verify"))
+            // Only the recipient can see the message
+            .ephemeral(true)
+    })
+    .await?;
```

Comment:
What does the user see if there are errors? How is that returned to them?

It would be great if we can actually log these errors that occur, using `tracing`.